js/script.js

```diff
@@ -0,0 +1,106 @@
+
+//creates an array of 9 elements based on the amount of divs on the page
+var boardSpace = document.getElementsByClassName("boardSpace");
+
+//starts off with playing X starting the game
+var whichPlayer = 'x';
+var isWinner = false;
+
+//listens for click on X div to close alert windows
+document.getElementById('exit').addEventListener("click", function(){
+	document.getElementById('alert').style.display = 'none';
+})
+
+//creates arrays for which spaces have been selected for X and O
+var selectedArrayX = [];
+var selectedArrayO = [];
```

Comment:
Storing the selected elements is a more efficient way for checking for answers, nice catch.